.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -46,12 +46,13 @@ steps:
   - script: |
```

Comment:
## [🕵 fluentuiv9 [New] Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=27585&commitId=d53a3c619b60adb02228d6268d139f67e6b5f591&env=prod&clientType=FLUENTUI)

_Diffs exceeded the limit allowed for comments. Please use the report link above to check your diffs._


<div id="vrtCommentE2E Tests-v9"/>